pypeit/flatfield.py

```diff
@@ -1355,10 +1233,9 @@ def show_flats(image_list, wcs_match=True, slits=None):
             clear = False
 
 
-def illum_profile_spectral(rawimg, waveimg, slits, model=None, gpmask=None, skymask=None, trim=3, flexure=None):
+def illum_profile_spectral(rawimg, waveimg, slits, ref_idx=0, smooth_npix=None, model=None, gpmask=None, skymask=None, trim=3, flexure=None):
     """
-    Generate a rough estimate of the relative spectral scaling of slits
-    using a low order polynomial. This routine is for slit-based IFUs.
+    Determine the relative spectral illumination of all slits.
```

Comment:
Can you please make it clear that this is only used for image slicing IFUs as in the previous documentation.

Reply:
Hmmm... Prof X and I discussed that this function might be used for correcting the vignetting in DEIMOS. So, I removed this from the docstring. I can take it out, provided this gets removed if it's used for DEIMOS. Given this, let me know what you prefer.

Reply:
Maybe just write "currently only used for IFUs" and then we can update it.